ci/copy_doxygen_files.sh

```diff
@@ -0,0 +1,31 @@
+#!/bin/sh
+
+# Prepare the source tree for Doxygen, see .github/workflows/doxygen.yml
+
+set -ex
+
+# Load the list of modified files.
+git fetch
+git branch -lr
+FILES=$(git diff --name-only origin/master... src/include)
```

Comment:
If you put the files into an array:

```suggestion
mapfile -t FILES < <(git diff --name-only origin/master... src/include)
```